internal/ingress/controller/controller.go

```diff
@@ -957,12 +964,14 @@ func (n *NGINXController) createServers(data []*ingress.Ingress,
 	}
 
 	// initialize default server and root location
+	ptf := networking.PathTypePrefix
 	servers[defServerName] = &ingress.Server{
 		Hostname: defServerName,
 		SSLCert:  n.getDefaultSSLCertificate(),
 		Locations: []*ingress.Location{
 			{
 				Path:         rootLocation,
+				PathType:     &ptf,
```

Comment:
Why not just use `networking.PathTypePrefix` directly here?

Reply:
Because we need a pointer